assets/app/view/game/part/city_slot.rb

```diff
@@ -39,7 +39,7 @@ def reservation
           h(
             :text,
             { attrs: { fill: 'black', transform: 'translate(0 9) scale(1.75)' } },
-            @reservation,
+            @reservation.sym,
```

Comment:
can just do reservation.name

Reply:
A company's `#name` is the full name:

![Screenshot from 2020-06-25 21-33-34](https://user-images.githubusercontent.com/1045173/85817582-946efd80-b72b-11ea-8686-3ecb20fd1b00.png)

I made an attempt at adding `Company#full_name` and moving `Company#sym` to `Company#name` to be consistent with `Corporation`, but kept running into the out of sync "It's not Player 1's turn" issue. https://github.com/michaeljb/18xx.games/commit/f3f318df2123847d2ef846a91931321f49592cd4 - [sample game JSON](https://gist.github.com/michaeljb/f6548901afa080dbacc67e5e29474123) (this game works on 3bbdbd1a, but doesn't work after merging in the latest master code, still looking into that)

I'm guessing I missed a spot where `#name` should be `#full_name`. Or could there be an issue with the MS/BIG4 minors having the same name as the companies? I'm not really sure what's going on there

Reply:
ah my bad i thought it was corporation.name not company. company it should be sym

Reply:
i thought @reservation was a corporation and not a company

Reply:
if it's both, you can do id

Reply:
>  (this game works on 3bbdbd1, but doesn't work after merging in the latest master code, still looking into that)

This fixed the issue I was seeing after merging latest master into this branch - 1c49aa07